Editor/Resources/EditorWindow/Components/ConnectToFleetInput.uxml

```diff
@@ -3,8 +3,8 @@
 
 <ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:gl="AmazonGameLift.Editor">
     <ui:Style src="../common.uss"/>
-    <ui:VisualElement name="AnywherePageConnectFleetStatusBoxContainer" class="separator"/>
-    <ui:VisualElement name="AnywherePageCreateFleet" class="separator">
+    <gl:StatusBox name="AnywherePageConnectFleetStatusBox"/>
+    <ui:VisualElement name="AnywherePageCreateFleet" class="separator separator--vertical">
```

Comment:
I probably don't understand css well enough, so please correct me if i'm wrong!

Is it not possible to set this on the parent and get it to propagate to it's children? I do believe we have css elements meant for this specific purpose, correct? That way you don't have to set it on every individual child


I probably don't understand css well enough, so please correct me if i'm wrong!

Is it not possible to set this on the parent and get it to propagate to it's children? I do believe we have css elements meant for this specific purpose, correct? That way you don't have to set it on every individual child

Reply:
Could you also rebase and resolve merge conflicts? The other PR has been merged in

Reply:
Merge conflicts resolved